### PR TITLE
RDBS-28 Old node tag should be kept as part of `GlobalStateSlice`

### PR DESCRIPTION
**Pull request by karmeli87**

### CODE REVIEW

File: src/Raven.Server/Rachis/RachisConsensus.cs [hunk 1]
```diff
@@ -462,6 +464,14 @@ public string ReadNodeTag(TransactionOperationContext context)
             return readResult == null ? InitialTag : readResult.Reader.ToStringValue();
         }
 
+        public string ReadPreviousNodeTag(TransactionOperationContext context)
```
Comment (ayende) [inline src/Raven.Server/Rachis/RachisConsensus.cs:467]: Can a server have _multiple_ old tags?

Comment (karmeli87) [reply src/Raven.Server/Rachis/RachisConsensus.cs:467]: You can call `HardResetToNewCluster` multiple times with different tag every time.

Comment (ayende) [reply src/Raven.Server/Rachis/RachisConsensus.cs:467]: Wouldn't that put us in the same situation?

Comment (karmeli87) [reply src/Raven.Server/Rachis/RachisConsensus.cs:467]: Shouldn't, because on every tag update we keep also the old one.

Comment (ayende) [reply src/Raven.Server/Rachis/RachisConsensus.cs:467]: ok

**Review by ayende — APPROVED** [review]

**Pull request merged**